Assignment-07-Product Designer/styles/css/assignment7.css

```diff
@@ -0,0 +1,150 @@
+body {
+  background-color: black;
+  margin: 0;
+  padding: 0;
+}
+
+.initial_header {
+  margin-top: 40px;
+  display: flex;
+  flex-direction: row-reverse;
+  color: #ffff;
+  margin-bottom: -8px;
+  font-family: "Roboto", "sans-serif";
+  font-weight: 600;
+  width: 88.4%;
+}
+
+.flex_container_shrutti {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  color: #ffff;
+  margin-top: 60px;
+}
+
+.shrutti {
+  font-size: 60px;
+  font-weight: 700;
+  font-family: "Roboto", "sans-serif";
+}
+
+.shrutti_first_name {
+  font-size: 60px;
+  font-weight: 300;
+  font-family: "Roboto", "sans-serif";
+}
+
+.designer {
+  font-size: 25px;
+  color: #ffff;
+  font-family: "Roboto", "sans-serif";
+  margin-top: 20px;
+}
+
+.product {
+  color: rgba(74, 144, 226, 255);
```

Comment:
Uniform color code properties

Reply:
uniform color code given.
- put # code instead of rgb.